docs/wa/missing_nncp.md

```diff
@@ -0,0 +1,72 @@
+# Missing NNCPs
+
+The following command defines the OpenStack control plane and its
+dependent CRs.
+```bash
+kustomize build architecture/examples/va/hci > control-plane.yaml
+```
+The `control-plane.yaml` file contains CRs for both `NMState` and
+`NodeNetworkConfigurationPolicy` (NNCP). When `oc apply -f` is passed
+this file, OpenShift might try to create the NNCPs while `NMState`
+CRDs are still installing and produce the following message.
+```
+nmstate.nmstate.io/nmstate created
+[resource mapping not found for name:
+"ostest-master-0" namespace: "openstack" from "control-plane.yaml":
+no matches for kind "NodeNetworkConfigurationPolicy" in version "nmstate.io/v1"
+ensure CRDs are installed first,
+resource mapping not found for name: "ostest-master-1" namespace: "openstack"
+from "control-plane.yaml": no matches for kind "NodeNetworkConfigurationPolicy"
+in version "nmstate.io/v1"
+```
+Retrying `oc apply -f contol-plane.yaml` a few seconds later should
+resolve the problem however.
+
+## Alternative Approach
+
+It's also possible to create CR files with less components and wait
+before applying each CR file. E.g. the file `nncp.yaml` would contain
+only `NodeNetworkConfigurationPolicy` CRs and `NMState` and other
+deployment related CRs could exist in another file like
+`deploy.yaml`. The following process may be used to do generate these
+files using kustomize.
+
+- Edit
+[va/hci/kustomization.yaml](https://github.com/openstack-k8s-operators/architecture/blob/main/va/hci/kustomization.yaml)
```

Comment:
URL paths are used here instead of relative paths because it's within the contents of the `docs` directory which can also be used to generate its own [mkdocs](https://www.mkdocs.org) page.

https://github.com/openstack-k8s-operators/architecture/blob/main/docs/contributing/documentation.md